packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,117 @@
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/grouping-mapping-widget.js",
+  "module": "lib/esm/grouping-mapping-widget.js",
+  "typings": "lib/cjs/grouping-mapping-widget",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": ""
+  },
+  "dependencies": {
+    "@itwin/imodel-browser-react": "^0.12.1",
+    "@itwin/itwinui-css": "^0.44.1",
+    "@itwin/itwinui-react": "^1.29.2",
+    "@itwin/itwinui-icons-react": "^1.5.1",
+    "classnames": "^2.3.1",
+    "cross-fetch": "^3.1.4",
+    "immer": "^9.0.7",
+    "simple-react-validator": "^1.6.1"
+  },
+  "devDependencies": {
+    "@itwin/appui-abstract": "^3.0.0",
+    "@itwin/appui-layout-react": "^3.0.0",
+    "@itwin/appui-react": "^3.0.0",
+    "@itwin/build-tools": "^3.0.0",
+    "@itwin/components-react": "^3.0.0",
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-common": "^3.0.0",
+    "@itwin/core-frontend": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/core-i18n": "^3.0.0",
+    "@itwin/core-markup": "^3.0.0",
+    "@itwin/core-orbitgt": "^3.0.0",
+    "@itwin/core-quantity": "^3.0.0",
+    "@itwin/core-react": "^3.0.0",
+    "@itwin/eslint-plugin": "^3.0.0",
+    "@itwin/presentation-common": "^3.0.0",
+    "@itwin/presentation-components": "^3.0.0",
+    "@itwin/presentation-frontend": "^3.0.0",
+    "@itwin/webgl-compatibility": "^3.0.0",
+    "@types/i18next": "^8.4.2",
+    "@types/node": "^14.14.20",
+    "@types/react": "^17.0.19",
+    "@types/react-dom": "^17.0.9",
+    "@types/react-redux": "^7.0.1",
+    "react-table": "^7.7.0",
+    "@types/react-table": "^7.7.0",
+    "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.10.1",
+    "cpx2": "^3.0.0",
+    "eslint": "^7.32.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-react": "^7.25.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.0",
+    "rimraf": "^3.0.2",
+    "typescript": "~4.3.0"
+  },
+  "peerDependencies": {
+    "@itwin/appui-abstract": "^3.0.0",
+    "@itwin/appui-react": "^3.0.0",
+    "@itwin/components-react": "^3.0.0",
```

Comment:
ton of imports from this pkg from outside the main barrel, please double check and fix it if possibile

if theyre not exported, theyre most likely internal apis which we dont really want to be using outside of the core pkgs

Reply:
I agree this is not good but for now it would take too long to reimplement the components that take advantage of this because we forked the property grid. We will make a backlog item for this.

Reply:
Please make sure the core team is aware that you need to use the internal apis, maybe theyll promote them to public

Reply:
@grigasp FYI that the property grid is currently being forked via this pkg.